content/docs/self-hosted-runners.md

```diff
@@ -437,17 +437,23 @@ provisioned through environment variables instead of files.
 
 The `cml runner` command can also be used to manually set up a local machine,
 on-premise GPU cluster, or any other cloud compute resource as a self-hosted
-runner. Simply [install CML](/doc/install) and then run:
+runner. Instead of having a `launch-runner` job (as in the YAML above), simply
+[install CML](/doc/install) on the local machine and then run:
```

Comment:
I agree that this paragraph could be rewritten to clarify that this is an alternative to the examples above but specifically referencing `launch-runner` isn't much clearer as that's just an arbitrary name.

Reply:
> BTW that name not used in the BitBucket setup example, which is confusing when you open that tab and read the next paragraph, "In the workflow above, the `launch-runner` step launches..." cc @0x2b3bfa0 @casperdcl

Reply:
Yeah, the point is that a user like me, who are not well versed in CML (nor runners TBH) did a bunch of experiments just to find out if that job was necessary or not for local runners. How about something like:


> runner. For local runners, you do not need to define a job to launch the runner (like the `launch-runner` in the example YAML above), you simply [install CML](/doc/install) on the local machine and then run:

Reply:
fixed in https://github.com/iterative/cml.dev/pull/417/files#r1046462973